src/App.tsx

```diff
@@ -8,6 +8,7 @@ function App(): JSX.Element {
   return (
     <div className="app">
       <header className="app-header">
+        <button className="btn btn-primary">Hello from Bootstrap</button>
```

Comment:
@gutakk Do we add this just to test that the Bootstrap is working? And are we going to remove it later?

Reply:
Yes, It will be removed when I start working on sign in page.